<ui version="4.0" >
 <class>SelectionDialog</class>
 <widget class="QDialog" name="SelectionDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Selection</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>4</number>
   </property>
   <property name="margin" >
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <item row="2" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Key</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Value</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLabel" name="label_3" >
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>=</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3" >
      <widget class="QLineEdit" name="edName" />
     </item>
     <item row="3" column="0" >
      <widget class="QComboBox" name="cbKey" />
     </item>
     <item row="3" column="2" >
      <widget class="QComboBox" name="cbValue" />
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Maximum returned results</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" >
      <widget class="QSpinBox" name="sbMaxResult" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>999</number>
       </property>
       <property name="value" >
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>493</width>
       <height>71</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelectionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
